box-share-sdk/src/main/java/com/box/androidsdk/share/views/CollaboratorsInitialsView.java

```diff
@@ -158,44 +159,50 @@ public void run() {
             };
 
 
-    public void updateUi(BoxIteratorCollaborations boxIteratorCollaborations) {
+    private void updateView(BoxIteratorCollaborations boxIteratorCollaborations) {
         mCollaborations = boxIteratorCollaborations;
-        if (mCollaborations != null && mCollaborations.size() != 0) {
+        if (mCollaborations != null) {
```

Comment:
For some reason when there are no collaborators i never got mCollaborations = null, i always got an empty collaborations collection. But I think adding the check will make it more robust in case the behavior changes in the future.